ctakesclient/client.py

```diff
@@ -39,13 +39,13 @@ def post(sentence: str, url: str = None) -> dict:
     """
     url = url or get_url_ctakes_rest()
     logging.debug(url)
-    # TODO: consider exposing a pass-through timeout parameter
-    response = requests.post(  # pylint: disable=missing-timeout
+    response = requests.post(
         url,
         data=sentence.encode("utf8"),
         headers={
             "Content-Type": "text/plain; charset=UTF-8",
         },
+        timeout=300,  # TODO: consider exposing a pass-through timeout parameter
```

Comment:
I considered doing this bit - adding an optional timeout parameter to a few methods. But my plan is still to revamp these methods to be async, instead of sync. So I'll just do it as part of that refactor. Not that much difference between 5 minutes and forever, really. You can quote me on that.

Reply:
am i remembering correctly that vlad's client has like a ten minute timeout? was there a reason?

Reply:
Hmm, I don't remember that number, but I could believe it. I picked 5 because the SMART FHIR authentication spec requires you to use a JWT timeout of no more than 5 minutes. These requetsts aren't against a FHIR server (they are ctakes and cnlpt requests), but I figured if FHIR isn't even giving overloaded EHR servers more than 5 minutes, no one needs more.